utils/src/main/java/com/cloud/utils/SwiftUtil.java

```diff
@@ -236,4 +247,60 @@ public static boolean deleteObject(SwiftClientCfg cfg, String path) {
         command.execute(parser);
         return true;
     }
+
+    public static boolean setTempKey(SwiftClientCfg cfg, String tempKey){
```

Comment:
I don't think I can write a unit test for this as it relies on an external script to work to give the correct output. I could mock the output but that would defeat the purpose of the unit test.